### DISCUSSION TITLE
Adding anchors on settings page

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#18837] [Feature Requests]
Seems interesting to adding anchors on all sections in settings/extensions and others settings pages. 

For example for a tutorial, I indicate to users to get their API key on https://dev.to/settings/extensions, but required to add a note like _go to DEV Community 👩‍💻👨‍💻 API Keys section_ for a better understanding.

**Comment by michael-tharrington** [comment]
Oooo yeah! I love this idea.

I'm frequently linking folks to different things in our settings, like publishing via RSS in https://dev.to/settings/extensions or editor version in https://dev.to/settings/customization ... it would be super cool to have a way to link folks directly to the thing!

**Comment by andypiper** [comment]
Definitely a nice UX improvement.
